zmsadmin/js/page/availabilityDay/form/conflicts.js

```
@@ -11,26 +11,31 @@ const renderConflictList = (conflictList) => {
-                    <div key={index}>- {key}</div>
-                </div>
-            )
-        })
-    )
-    
-    
-}
+        conflictDatesByMessage.map((item, index) => (
+            <div key={index} style={{ marginBottom: '1rem' }}>
+                {/* Convert newlines in the message to <br /> tags */}
+                <div dangerouslySetInnerHTML={{ __html: item.message.replace(/\n/g, '<br />') }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)